client/ayon_houdini/plugins/publish/collect_render_products.py

```diff
@@ -168,6 +169,15 @@ def get_aov_identifier(self, render_product):
             str: The AOV identifier
 
         """
+        # Allow explicit naming through custom attribute on the render product
+        attr = render_product.GetPrim().GetAttribute("ayon:aov_identifier")
+        if attr:
+            aov_identifier = attr.Get(time=Usd.TimeCode.EarliestTime())
+            self.log.debug(
+                "Using explicit ayon:aov_identifier on render product"
+                f" '{render_product}': '{aov_identifier}'")
```

Comment:
How about this @MustafaJafar ?
```suggestion
                "Using explicit aov identifier on render product"
                f" '{attr.GetPath()}': '{aov_identifier}'")
```

Reply:
using `attr.GetPath()` -> `'/Render/Products/renderproduct.ayon:aov_identifier'`
using `render_product.GetPath()` -> `'/Render/Products/renderproduct'`

Reply:
```suggestion
                "Using explicit ayon:aov_identifier on render product"
                f" '{render_product.GetPath()}': '{aov_identifier}'")
```

Reply:
Thanks - I prefer it without the dot in this case actually.